src/main/java/emu/grasscutter/server/game/GameSession.java

```diff
@@ -122,18 +123,18 @@ public void send(BasePacket packet) {
         // Log
         switch (GAME_INFO.logPackets) {
             case ALL -> {
-                if (!PacketOpcodesUtils.LOOP_PACKETS.contains(packet.getOpcode())) {
-                    logPacket("SEND", packet.getOpcode(), packet.getData());
+                if (!PacketOpcodesUtils.LOOP_PACKETS.contains(packet.getOpcode()) || GAME_INFO.isShowLoopPackets) {
+                    logPacket("SEND [ALL]", packet.getOpcode(), packet.getData());
```

Comment:
Is there a point to reminding the reader what the filter is set to with every message?

Reply:
Yes, this is exactly so, I agree it is not entirely useful, so I can remove it, should I do this?

Reply:
Yes please

Reply:
Removed